mreg_cli/policy.py

```diff
@@ -597,7 +597,7 @@ def host_remove(args):
         path = f'/api/v1/hostpolicy/roles/{args.role}/hosts/{name}'
         history.record_delete(path, dict())
         delete(path)
-        cli_info(f"Removed '{name}' from {args.role}", print_msg=True)
+        cli_info(f"Removed host {name} from role {args.role}", print_msg=True)
```

Comment:
Should we quote the variables to make them stand out a bit better? Ala:
`cli_info(f"Added host '{name}' to role '{args.role}'", print_msg=True)`